frontend/src/metabase/parameters/utils/parameter-type.ts

```diff
@@ -1,15 +1,20 @@
-import { Parameter } from "metabase-types/api";
 import { SINGLE_OR_MULTI_SELECTABLE_TYPES } from "metabase-lib/parameters/constants";
 import {
   getParameterSubType,
   getParameterType,
 } from "metabase-lib/parameters/utils/parameter-type";
+import type { ParameterWithTemplateTagTarget } from "metabase-lib/parameters/types";
 
-export function isSingleOrMultiSelectable(parameter: Parameter): boolean {
+export function isSingleOrMultiSelectable(
+  parameter: ParameterWithTemplateTagTarget,
```

Comment:
`Parameter` is a type for our API, and `hasVariableTemplateTagTarget` only exist client-side.